server.go

```diff
@@ -2218,9 +2252,7 @@ func writeResponse(ctx *RequestCtx, w *bufio.Writer) error {
 	if ctx.timeoutResponse != nil {
 		panic("BUG: cannot write timed out response")
 	}
-	err := ctx.Response.Write(w)
-	ctx.Response.Reset()
```

Comment:
I see you moved the `Response.Reset()` to the callers of `writeResponse` but you didn't do this for all callers causing some bugs.

Reply:
Thanks, updated.